src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Dec2Hex.cs

```diff
@@ -0,0 +1,214 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.PowerFx.Core.App.ErrorContainers;
+using Microsoft.PowerFx.Core.Binding;
+using Microsoft.PowerFx.Core.Errors;
+using Microsoft.PowerFx.Core.Functions;
+using Microsoft.PowerFx.Core.Localization;
+using Microsoft.PowerFx.Core.Types;
+using Microsoft.PowerFx.Core.Utils;
+using Microsoft.PowerFx.Syntax;
+
+namespace Microsoft.PowerFx.Core.Texl.Builtins
+{
+    // Dec2Hex(number:n, [places:n])
+    internal sealed class Dec2HexFunction : BuiltinFunction
+    {
+        public override bool IsSelfContained => true;
+
+        public override bool IsStateless => true;
+
+        public override bool SupportsParamCoercion => true;
+
+        public override bool HasPreciseErrors => true;
```

Comment:
>public override bool HasPreciseErrors => true; [](http://example.com/codeflow?start=8&length=46)

I believe if we don't override CheckInvocation (and provide the errors ourselves) we don't need to define this.

Reply:
it requires the override otherwise this error shows up
'Dec2HexTFunction.CheckInvocation(CheckTypesContext, TexlNode[], DType[], IErrorContainer, out DType, out Dictionary<TexlNode, DType>)': new protected member declared in sealed type

Reply:
I'm saying that we probably don't need the method at all (not just the `override` keyword) - if we specify the parameter types in the constructor (`...DType.String, 0, 1, 2, DType.Number, DType.Number)`)